src/generic_ui/polymer/cloud-install.ts

```diff
@@ -35,8 +35,8 @@ Polymer({
     ui.cloudUpdate({
       operation: uproxy_core_api.CloudOperationType.CLOUD_HAS_OAUTH,
       providerName: DEFAULT_PROVIDER
-    }).then((hasOAuth :boolean) => {
-      if (hasOAuth) {
+    }).then((result :uproxy_core_api.CloudOperationResult) => {
+      if (result.hasOAuth) {
```

Comment:
@trevj Missed this in c54dc0a but caught it in manual testing, yikes that was close!